src/components/Form.js

```diff
@@ -251,13 +253,18 @@ class Form extends React.Component {
       DonneesDescription,
       availableScopes,
       AdditionalRgpdAgreement,
-      AdditionalDataContent,
       AdditionalCguContent,
       AdditionalContent,
     } = this.props;
 
     const disabledApplication = !acl.send_application;
 
+    const groupTitleScopes = availableScopes.filter(scope => scope.groupTitle);
```

Comment:
To my opinion, this step can be avoided, the following `groupBy` call applied on `availableScopes` will create an "undefined" group that can be treated as the general group. This can prevent treating the general group differently than the other groups :tada:

less is more !